app.py

```diff
@@ -1,9 +1,45 @@
+import os
 from flask import Flask
-app = Flask(__name__)
+from models import db, connect_db
+from flask_sqlalchemy import SQLAlchemy
 
-@app.route('/')
-def hello_world():
-    return 'Hello, World!'
+from dotenv import load_dotenv
+load_dotenv()
+
+# some default test values until we use dotenv
+DBUSER = os.getenv('POSTGRES_USER')
```

Comment:
heroku only takes `DATABASE_URL` so we don't actually need to specify `HOST` `PORT` `DB` etc